static/app/views/performance/newTraceDetails/traceMetrics.tsx

```diff
@@ -60,38 +58,30 @@ export function TraceViewMetricsDataProvider({
     queryParams.statsPeriod,
   ]);
 
-  return (
-    <SingleMetricQueryParamsProvider>
-      <TraceViewMetricsDataProviderInner traceSlug={traceSlug} tracePeriod={tracePeriod}>
-        {children}
-      </TraceViewMetricsDataProviderInner>
-    </SingleMetricQueryParamsProvider>
-  );
-}
-
-function TraceViewMetricsDataProviderInner({
-  children,
-  traceSlug,
-  tracePeriod,
-}: {
-  children: React.ReactNode;
-  traceSlug: string;
-  tracePeriod?: TracePeriod;
-}) {
-  const {queryParams, setQueryParams, metric, setTraceMetric, removeMetric} =
-    useSingleMetricQueryParams();
-
   return (
     <MetricsQueryParamsProvider
-      queryParams={queryParams}
-      setQueryParams={setQueryParams}
-      traceMetric={metric}
-      setTraceMetric={setTraceMetric}
-      removeMetric={removeMetric}
+      queryParams={
+        new ReadableQueryParams({
+          extrapolate: true,
+          mode: Mode.SAMPLES,
+          query: '',
+          cursor: '',
+          fields: ['id', 'timestamp'],
+          sortBys: [{field: 'timestamp', kind: 'desc'}],
+          aggregateCursor: '',
+          aggregateFields: [],
+          aggregateSortBys: [],
+        })
+      }
+      setQueryParams={() => {}}
+      traceMetric={{name: '', type: ''}}
+      setTraceMetric={() => {}}
+      removeMetric={() => {}}
```

Comment:
The signature is a little odd here as it forces you to pass these empty values but this is fine for now I guess.